npm/qsharp/src/language-service/language-service.ts

```diff
@@ -117,6 +118,7 @@ export class QSharpLanguageService implements ILanguageService {
       readFile: async () => null,
       listDirectory: async () => [],
       resolvePath: async () => null,
+      fetchGithub: async () => "",
```

Comment:
Note from Mine's past self: would it make sense to provide a default implementation that the playground can also use?

Reply:
Reply from my current self: Yes, that would be useful, but I don't have the time right now :)